golang/server/posts.go

```diff
@@ -0,0 +1,92 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/karlkeefer/pngr/golang/env"
+	"github.com/karlkeefer/pngr/golang/errors"
+	"github.com/karlkeefer/pngr/golang/models"
+	"github.com/karlkeefer/pngr/golang/server/write"
+)
+
+// helpers for easily parsing params
+func getID(r *http.Request) (id int64, err error) {
```

Comment:
probably we want getInt(name) and getString(name) that lives in server.go, not `posts.go`